Sources/Controllers/2.0/OATargetInfoViewController.mm

```diff
@@ -555,7 +557,10 @@ - (CGFloat) tableView:(UITableView *)tableView heightForFooterInSection:(NSInteg
 - (CGFloat) tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
 {
     OARowInfo *info = _rows[indexPath.row];
-    return info.height;
```

Comment:
**info.height** is used while calculating contentHeight (for correct layout and scroll the menu). Looks like such fix broke contentHeight calculation. Thus menu will not be scrolled correctly to the most bottom row.